### PR TITLE
Add external dependencies UI

### PR DESCRIPTION
**Pull request by mayankkapoor**
Now the user can add external dependencies via UI.

### CODE REVIEW

File: aasemble/django/apps/buildsvc/views.py [hunk 1]
```diff
@@ -44,6 +45,40 @@ def package_source(request, source_id):
...
+        form = get_external_dependency_form(request, request.POST, instance=dependency)
+
+        if dependency is not None and request.POST.get('delete', '') == 'delete':
+            dependency.delete()
+            return HttpResponseRedirect(reverse('buildsvc:external_dependencies'))
+
+        if ((form.is_valid() and
+             form.cleaned_data['own_series'].user_can_modify(request.user))):
+            form.save()
+            print("new_dependency saved")
```
Comment (sorenisanerd) [inline aasemble/django/apps/buildsvc/views.py]: Can you remove this print call, please?

**Comment by sorenh** [comment]
Does the [WIP] imply that I shouldn't review/merge it yet?

**Comment by sorenisanerd** [comment]
@mayankkapoor Does the "[WIP]" imply that I shouldn't review/merge this yet?

**Comment by mayankkapoor** [comment]
@sorenh I'll remove the WIP once the PR #184 is merged. I think I'll need to rebase once #184 is merged. You can still review this and give feedback.

**Pull request merged**